packages/editor/src/components/hierarchy/HierarchyPanelContainer.tsx

```diff
@@ -224,9 +223,9 @@ function HierarchyPanelContents({ rootEntityUUID }: { rootEntityUUID: EntityUUID
 
   const onClick = useCallback((e: MouseEvent, node: HeirarchyTreeNodeType) => {
     if (e.detail === 2) {
-      const editorCameraState = getMutableState(EditorCameraState)
-      editorCameraState.focusedObjects.set([node.entity])
-      editorCameraState.refocus.set(true)
+      //const editorCameraState = getMutableState(EditorCameraState)
```

Comment:
should this be re-enabled?

Reply:
Yes. Forgot about this bit, just rewrote it.